app/views/moderator_applications/show.html.slim

```diff
@@ -0,0 +1,38 @@
+.container-fluid
+  .row
+    .col-lg-2
+
+    main.restore-defaults.col-lg-8
+      h1
+        = link_to(@application.user.username, user_path(@application.user))
+        | 's Moderator Application
+
+      - if current_user.is_admin?
+        a.btn.btn-primary href=moderator_applications_path style='margin-right: 5px;'
+          span.fa.fa-undo Back
```

Comment:
Would a non-admin back button make sense? Probably not, but I still don't know exactly what the flow for back buttons on this site is. (And this is an unrelated issue, but it can be confusing when you're browsing a user's submissions page, view a submission, hit the back button, and end up on the submissions index. *Especially* since it's not even the page that submission is on! (That part in particular should be considered a bug, maybe?)